src/sentry/api/endpoints/organization_integration_repository_project_path_config_details.py

```diff
@@ -48,11 +58,13 @@ def put(
                 serialize(repository_project_path_config, request.user),
                 status=status.HTTP_200_OK,
             )
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
-        return self.respond(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    def delete(self, request, config_id, organization, config):
+        """
+        Delete a repository project path config
 
-    def delete(
-        self, request, organization_slug, integration_id, organization, org_integration, config
-    ):
+        :auth: required
+        """
         config.delete()
-        return self.respond(status=status.HTTP_204_NO_CONTENT)
+        return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
@NisanthanNanthakumar why change `self.respond` to `Response`?

Reply:
@scefali ah its the same!

Reply:
@NisanthanNanthakumar no need, was just curious